pkg/resources/view_acceptance_test.go

```diff
@@ -25,44 +25,57 @@ import (
 )
 
 // TODO(SNOW-1423486): Fix using warehouse in all tests and remove unsetting testenvs.ConfigureClientOnce
-// TODO(next pr): cleanup setting warehouse with unsafe_execute
 func TestAcc_View_basic(t *testing.T) {
 	t.Setenv(string(testenvs.ConfigureClientOnce), "")
 	_ = testenvs.GetOrSkipTest(t, testenvs.EnableAcceptance)
 	acc.TestAccPreCheck(t)
 
-	rowAccessPolicy, rowAccessPolicyCleanup := acc.TestClient().RowAccessPolicy.CreateRowAccessPolicyWithDataType(t, sdk.DataTypeVARCHAR)
+	rowAccessPolicy, rowAccessPolicyCleanup := acc.TestClient().RowAccessPolicy.CreateRowAccessPolicyWithDataType(t, sdk.DataTypeNumber)
 	t.Cleanup(rowAccessPolicyCleanup)
 
 	aggregationPolicy, aggregationPolicyCleanup := acc.TestClient().AggregationPolicy.CreateAggregationPolicy(t)
 	t.Cleanup(aggregationPolicyCleanup)
 
-	rowAccessPolicy2, rowAccessPolicy2Cleanup := acc.TestClient().RowAccessPolicy.CreateRowAccessPolicyWithDataType(t, sdk.DataTypeVARCHAR)
+	rowAccessPolicy2, rowAccessPolicy2Cleanup := acc.TestClient().RowAccessPolicy.CreateRowAccessPolicyWithDataType(t, sdk.DataTypeNumber)
 	t.Cleanup(rowAccessPolicy2Cleanup)
 
 	aggregationPolicy2, aggregationPolicy2Cleanup := acc.TestClient().AggregationPolicy.CreateAggregationPolicy(t)
 	t.Cleanup(aggregationPolicy2Cleanup)
 
+	functionId := sdk.NewSchemaObjectIdentifier("SNOWFLAKE", "CORE", "AVG")
```

Comment:
nit: we could extract these (like we did with snowflake preexisting roles)

Reply:
Yes, but it's an arbitrarily chosen function with no specific meaning, and we use it only in tests, so I'll leave this for now.

Reply:
oky